Source/Tools/IRLoader/CMakeLists.txt

```
@@ -0,0 +1,25 @@
+list(APPEND LIBS FEXCore Common)
+
+set (DEFINES)
+if (ENABLE_VIXL_SIMULATOR)
+  list(APPEND DEFINES -DVIXL_SIMULATOR=1)
+endif()
+
+add_executable(IRLoader
+  IRLoader.cpp
+)
+target_compile_definitions(IRLoader PRIVATE ${DEFINES})
```

Dropping use of the variable keeps control flow straightforward:

```suggestion
if (ENABLE_VIXL_SIMULATOR)
    target_compile_definitions(IRLoader PRIVATE "-DVIXL_SIMULATOR=1")
endif()
```